### ISSUE TITLE
Usability Enhancement: IPython REPR errors for solids/faces/edges/etc

### ISSUE THREAD

**Issue by jdegenstein** [gumyr/build123d#89]
I noticed some minor strange behavior with the default repr for solids/faces/edges with build123d and the ipython console in CQ-editor. With this example part:
```py
from build123d import *
length, width, thickness = 80.0, 60.0, 10.0
with BuildPart() as ex1:
    Box(length, width, thickness)
```
If you then execute `ex1.part` in the ipython console, you get the following:
```
In [9]: ex1.part
---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
File ~\.conda\envs\build123ddev\Lib\site-packages\IPython\core\formatters.py:343, in BaseFormatter.__call__(self, obj)
    341     method = get_real_method(obj, self.print_method)
    342     if method is not None:
--> 343         return method()
    344     return None
    345 else:

File ~\.conda\envs\build123ddev\Lib\site-packages\build123d\direct_api.py:3183, in Shape._repr_javascript_(self)
   3180 def _repr_javascript_(self):
   3181     """Jupyter 3D representation support"""
-> 3183     from .jupyter_tools import display
   3185     return display(self)._repr_javascript_()

File ~\.conda\envs\build123ddev\Lib\site-packages\build123d\jupyter_tools.py:7, in <module>
      2 from json import dumps
      4 from IPython.display import Javascript
----> 7 from .exporters.vtk import toString
      8 from .shapes import Shape
      9 from ..assembly import Assembly

ModuleNotFoundError: No module named 'build123d.exporters'
Out[9]: <build123d.direct_api.Solid at 0x24d92c089a0>
```
I find the same behavior with parts/solids/sketches/faces/lines/wires/edges. I have noticed that log/print produces normal output (similar to the last line of the ipython output).

**Comment by jdegenstein** [comment]
Closed as it was fixed by #109

**Comment by gumyr** [comment]
@jdegenstein Thanks for looking after this - greatly appreciated!
